src/machine_data_hub/cli.py

```diff
@@ -19,49 +21,97 @@ def get_datasets(url):
+            files_list.append(file_and_size)
+        #num = len(row["Datasets"])
+        #sizes = [file["File Size"] for file in row["Datasets"]]
+        #output = id + " " + name + " (" + num + " files, "
+        #x = ', '.join(sizes)
+        #output += x
+        #output += ")"
+        #list_data.append(output)
+        data_info["files"] = files_list
+        list_data.append(data_info)
+    return list_data
+
+def dataset_ids(datasets):
+    ids = [int(row['id']) for row in datasets]
+    return ids
 
 
 @app.command("suggest")
-def suggest(link: str = typer.Option(..., prompt="Please enter the link to the dataset you want to suggest"),
-            name: str = typer.Option(..., prompt="Please enter a name for the dataset at the link you provided"),
-            summary: str = typer.Option(..., prompt="Please enter a short summary describing the dataset"),):
+def suggest(name: str, link: str, summary: str):
     """Suggest a dataset to be added to the Machine Data Hub by giving a link, name, and summary. """
-    typer.echo(f"{SUGGESTION_FILE}")
-    with open(SUGGESTION_FILE, mode="a") as suggestions_file:
-        employee_writer = csv.writer(
-            suggestions_file, delimiter=",", quotechar='"', quoting=csv.QUOTE_MINIMAL
-        )
-        employee_writer.writerow([name, summary, link])
+    org = "PHM-Data-Hub"
+    team_slug = "uw-capstone-team"
+    discussion_number = 1
+    date_time = str(datetime.date.today()) + " " + str(datetime.datetime.now().time())
+    body = " ### " + name + "\n" + date_time + "\n\n**Summary:** " + summary + "\n**Link:** " + link + "\n\nSubmitted from command line interface"
+    query_url = f"https://api.github.com/orgs/{org}/teams/{team_slug}/discussions/{discussion_number}/comments"
+    data = {
+        "body": body
+    }
+    headers = {'Authorization': f'token {TOKEN}'}
+    r = requests.post(query_url, headers=headers, data=json.dumps(data))
     typer.echo(
         f"Thank you! You have suggested a dataset, {name}, from the following link: {link}"
     )
 
 
 @app.command("download")
-def download(name: str):
+def download(id: int, file: int = typer.Argument(None)):
     """Download a dataset by passing in the name. """
     datasets = get_datasets(API_URL)
-    if name in dataset_names(datasets):
-        typer.echo(f"Downloading {name} right now!")
-        url = [row["DownloadLink"] for row in datasets if row["Name"] == name][0]
-        r = requests.get(url, allow_redirects=True)
-        # save content with name
-        with open(f"{name}", "wb") as fid:
-            fid.write(r.content)
+
+    # if dataset id exists
+    if id in dataset_ids(datasets):
+        # loop through each dataset
+        for row in datasets:
+            # find the dataset with given id
+            if int(row["id"]) == id:
+                # once found, saving name of dataset
+                name = row["Name"]
+                # if optional argument is passed
+                if file:
```

Comment:
suggest changing to `if not (file is None):` to be explicit about checking for None. Otherwise you have to deal with "truthiness" which can have intended consequences. for example if you did label the first file as a `0` that would not be `None` but this if statement would still fail to trigger.